frontend/notification/notifications.html

```diff
@@ -17,7 +17,7 @@
             <img ng-src="{{notification.from.photo_url}}" class="md-avatar" style="width: 60px; height: 60px;"/>
             <div class="md-list-item-text" layout="column" >
               <p><b style="font-size: 18px;">{{ notificationCtrl.limitString(notification.from.name, 28) }}</b></p>
-              <p><b style="font-size: 15px; color: #009688;">{{ notificationCtrl.limitString(notification.current_institution.name, 32) }}</b></p>
+              <p><b style="font-size: 15px; color: #009688;">{{ notificationCtrl.limitString(notification.current_institution.name, 32) || notificationCtrl.limitString(notification.from.institution_name, 32)}}</b></p>
```

Comment:
insert a comment explaining what is new notification data and what is old notification data